services/docker-compose.yml

```diff
@@ -1,5 +1,35 @@
-version: '2'
```

Comment:
There is a junja2 template version of this our other repo. Which should we use?

Reply:
What docker-compose file is this? The one that gets deployed is produced by Ansible I believe?